service/src/registry.ts

```diff
@@ -624,7 +624,7 @@ export class Registry {
       const path = '/twins/' + encodeURIComponent(deviceId) + endpoint.versionQueryString();
       const headers = {
         'Content-Type': 'application/json; charset=utf-8',
-        'If-Match': etag
+        'If-Match': '"' + etag + '"'
```

Comment:
Adjusted so that if the first and last char are " then it simply leaves the etag alone.  Otherwise it will encase in "

---
In reply to: [318814757](https://github.com/Azure/azure-iot-sdk-node/pull/613#discussion_r318814757) [](ancestors = 318814757,318413477)